src/inspector_agent.h

```diff
@@ -33,6 +35,9 @@ class Agent {
   // Stop the inspector agent
   void Stop();
 
+  void ContextCreated(const v8_inspector::V8ContextInfo& info);
```

Comment:
I believe it would be better if this call accepted v8::Context (and, optionally, a string) rather then V8ContextInfo. This would allow us no to leak inspector API to the bigger Node.